src/rest/RequestHandler.js

```diff
@@ -0,0 +1,174 @@
+const DiscordAPIError = require('./DiscordAPIError');
+const HTTPError = require('./HTTPError');
+const Util = require('../util/Util');
+const { Events: { RATE_LIMIT }, browser } = require('../util/Constants');
+
+function parseResponse(res) {
+  if (res.headers.get('content-type').startsWith('application/json')) return res.json();
+  // eslint-disable-next-line no-undef
+  if (browser) return res.blob();
```

Comment:
why the eslint disable

Reply:
My eslint was acting up, it seems, will remove it.